scrape.py

```diff
@@ -57,6 +58,12 @@ def get_listing_description(listing: Tag, data: dict):
     # Return the list as a string, joined by newline characters
     return "\n".join(children_list)
 
+def get_category(title: str):
```

Comment:
This is perfectly fine (assuming each listing will only ever have one category) but in the future look into functions like filter, map, reduce, and next for doing this kind of thing with lists